core/src/main/java/fr/sncf/osrd/speedcontroller/generators/LinearAllowanceGenerator.java

```diff
@@ -28,7 +26,7 @@ public LinearAllowanceGenerator(double begin, double end,
                                          Set<SpeedController> maxSpeeds) {
         // find the percentage of the allowance to add to the whole path
         double percentage;
-        if (allowanceType.equals(MarginType.TIME))
+        if (allowanceType.equals(MarginType.PERCENTAGE))
```

Comment:
The case `allowanceType == TIME` is not handle anymore.

Reply:
This is the linearAllowanceGenerator so it shouldn't be called anymore, but i'll quickly change it in case of...